gm4_better_armor_stands/data/better_armor_stands/functions/update.mcfunction

```diff
@@ -0,0 +1,25 @@
+@s = written book ..1 from armor_stand
```

Comment:
I think this function would serve a rename. Update is already used in the installation procedures and is a bit descriptive here.. Maybe something like "book_apply" or "stand_edit" would be more descriptive and appropriate.